tgui/packages/tgui/interfaces/ProcessingConsole.tsx

```diff
@@ -0,0 +1,149 @@
+import { toTitleCase } from 'common/string';
+
+import { useBackend } from '../backend';
+import {
+  Button,
+  Icon,
+  Image,
+  NoticeBox,
+  Section,
+  Stack,
+  Table,
+} from '../components';
+import { Window } from '../layouts';
+import { Material } from './Fabrication/Types';
+
+type IconData = {
+  id: string;
+  icon: string;
+};
+
+type Alloy = {
+  name: string;
+  id: string;
+};
+
+type ProcessingConsoleProps = {
```

Comment:
Just "Data" preferably for local backend data

Reply:
How should I name props for internal components then? I'm just curious.

Reply:
I think it's ok to distinguish multiple prop types in one file using `ComponentNameProps` format, but for backend data it's always just Data (there is only ever one anyways*)

*Unless you're wanting to export it, like in the case of NTOS screens with shared backend data types